packages/babel-preset-stage-0/README.md

```diff
@@ -48,11 +48,18 @@ module.exports = function() {
     plugins: [
       require("@babel/plugin-syntax-dynamic-import"),
       [require("@babel/plugin-proposal-decorators"), { "legacy": true }],
-      [require("@babel/plugin-proposal-class-properties"), { "loose": false }],
+      [require("@babel/plugin-proposal-class-properties"), { "loose": true }],
     ],
     presets: [
       // ...
     ],
   };
 };
 ```
+
+**NOTE: Compatibility with @babel/plugin-proposal-class-properties**
```

Comment:
```suggestion
**NOTE: Compatibility between `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-decorators`**
```

Reply:
I will apply these changes for all, right?